driver_scripts/setup_ice_shelves.py

```diff
@@ -0,0 +1,198 @@
+#!/usr/bin/env python
+
+"""
+This script creates region groups for ice shelves
+
```

Comment:
I don't know if it is worth noting in the comments here that this script needs to be run from the root level directory of 'geometric_features'? Perhaps that is already implied by all of the scripts that sit in the 'driver_scripts' subdir?